src/main/java/service/operation/OperationStrategyImpl.java

```diff
@@ -12,6 +12,12 @@ public OperationStrategyImpl(Map<Transaction.Operation, OperationHandler> operat
 
     @Override
     public OperationHandler getOperation(Transaction transaction) {
-        return operationsMap.get(transaction.getOperation());
+        OperationHandler handler = operationsMap.get(transaction.getOperation());
+        if (handler == null) {
+            throw new RuntimeException("There is no handler for transaction "
+                    + transaction.getOperation());
+        } else {
+            return handler;
+        }
```

Comment:
```suggestion
            return handler;
```
no else after throwing/returning is needed